app/searches/lookup_related_drugs.rb

```diff
@@ -3,12 +3,11 @@ class LookupRelatedDrugs
   def self.find(drug_name)
     drug_name = drug_name.gsub('(', '\(').gsub(')', '\)').gsub(':', '\:')
     drugs = DataModel::Drug.advanced_search(name: drug_name)
+    #Can we get rid of thise case?
```

Comment:
Same as previous comment re: GeneClaim. I think it's good for us to move away from searching against claims. Ideally, we want them to appear only in the full interactions.tsv dump for now.